docs/resource/oci.md

```diff
@@ -0,0 +1,15 @@
+# OCI
```

Comment:
Should this be a subsection under `container.md`?  I.e. OCI specifics?

Reply:
+1 on moving this markdown section under `container.md`. 
Actually, another good example in which [decoupling attribute definition from usage would help](https://github.com/open-telemetry/semantic-conventions/issues/197).